packages/core/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "abitype": "^0.8.3",
     "eventemitter3": "^5.0.1",
-    "viem": "^2.5.0",
+    "viem": "^2.7.8",
```

Comment:
viem versions should be updated all packages together not just aa-core. let's revert this for now unless not required.